### PR TITLE
Updated Readme.md

### PR DESCRIPTION
**Pull request by sudharshan-govindan**
Hi @nikhita : please review and approve the changes at the earliest !

### CODE REVIEW

**Comment by nikhita** [comment]
@sudharshan-govindan wow, thanks for adding so many conferences! Just wanted to let you know that I've seen this PR and will review it by today. :+1:

**Comment by sudharshan-govindan** [comment]
@nikhita : Thanks for your lightning response !

I am a Developer Advocate in IBM, Bengaluru. We use your repo to communicate to our developers about the list of conferences, workshops etc happening through out the year.

Fantastic initiative 👍

**Comment by aravindputrevu** [comment]
@sudharshan-govindan Awesome 👍 Great list of conferences to know. 
But, most of them are not having an option to submit a CFP. And one devops conference is from a company. 

@nikhita May be we should create a field separating community and organization driven conferences?

**Comment by sudharshan-govindan** [comment]
Hi Admin(s),

Can you please review and approve the PR at the earliest ?

**Comment by nikhita** [comment]
>  But, most of them are not having an option to submit a CFP. And one devops conference is from a company.

Thanks for pointing this out, I was not aware of this. :+1: 

> @nikhita May be we should create a field separating community and organization driven conferences?

I would rather remove the organization driven conferences. I don't want to include commercially driven conferences in this repo. It opens a huge can of worms, which I don't want to deal with. :)

If someone wants to include them, they are free to open a new repo for it. 

> Can you please review and approve the PR at the earliest ?

Sorry, I'm busy at $DAYJOB but I'll get to it soon.

File: README.md [hunk 1]
```diff
@@ -10,14 +10,31 @@ To add a conference to the list, please [contribute](#contributing)!
 |------------|------|-------|-------------|
 | [FunctionalConf](https://functionalconf.com/) | 12-16 Dec | The LaLiT Ashok Hotel, Bengaluru | Bring together the growing community of functional programmers under one roof. |
 | [Devops Days India](http://devopsdaysindia.org/) | 8-9 Dec | Sheraton Grand, Bengaluru | Devopsdays is a worldwide series of technical conferences covering topics of software development, IT infrastructure operations, and the intersection between them. | 
+| [DataHack Summit](https://www.analyticsvidhya.com/datahack-summit-2018/) | 22-24 Nov | NIMHANS Convention Center, Bengaluru | Conference On Artificial Intelligence, Machine Learning, Deep Learning & IOT | 
+| [NASSCOM Product Conclave](http://productconclave.in/) | 25-26 Oct | Vivanta by Taj, Bengaluru | NPC has always been synonymous with the amazing entrepreneurial ecosystem of innovative software products developed, engineered, marketed and sold from India and the globe. NPC, this year will discuss how to creatively enhance businesses when the next generation of technologies, including machine intelligence, is everywhere ! |
```
Comment (nikhita) [inline README.md]: Please remove this conference since this does not have a CFP. If it has, I'm happy to be pointed to it. :)

File: README.md [hunk 2]
```diff
@@ -10,14 +10,31 @@ To add a conference to the list, please [contribute](#contributing)!
...
 | [Devops Days India](http://devopsdaysindia.org/) | 8-9 Dec | Sheraton Grand, Bengaluru | Devopsdays is a worldwide series of technical conferences covering topics of software development, IT infrastructure operations, and the intersection between them. | 
+| [DataHack Summit](https://www.analyticsvidhya.com/datahack-summit-2018/) | 22-24 Nov | NIMHANS Convention Center, Bengaluru | Conference On Artificial Intelligence, Machine Learning, Deep Learning & IOT | 
+| [NASSCOM Product Conclave](http://productconclave.in/) | 25-26 Oct | Vivanta by Taj, Bengaluru | NPC has always been synonymous with the amazing entrepreneurial ecosystem of innovative software products developed, engineered, marketed and sold from India and the globe. NPC, this year will discuss how to creatively enhance businesses when the next generation of technologies, including machine intelligence, is everywhere ! | 
+| [MODS](http://www.developermarch.com/mds/index.html) | Oct | Indian Institute of Science, Bengaluru | Mobile Developer Summit is the biggest and definitive multi-platform conference and expo series for India's mobile developer ecosystem. |
```
Comment (nikhita) [inline README.md]: Please remove this conference since this does not have a CFP.

File: README.md [hunk 3]
```diff
@@ -10,14 +10,31 @@ To add a conference to the list, please [contribute](#contributing)!
...
+| [NASSCOM Product Conclave](http://productconclave.in/) | 25-26 Oct | Vivanta by Taj, Bengaluru | NPC has always been synonymous with the amazing entrepreneurial ecosystem of innovative software products developed, engineered, marketed and sold from India and the globe. NPC, this year will discuss how to creatively enhance businesses when the next generation of technologies, including machine intelligence, is everywhere ! | 
+| [MODS](http://www.developermarch.com/mds/index.html) | Oct | Indian Institute of Science, Bengaluru | Mobile Developer Summit is the biggest and definitive multi-platform conference and expo series for India's mobile developer ecosystem. | 
+| [OSI](https://www.opensourceindia.in/) | 11-12 Oct | NIMHANS Convention Center, Bengaluru | Started as LinuxAsia in 2004, OSI has been at the helm of bringing together the Open Source industry and the community in the last 14 years. The 15th edition of OSI this year aims to take this event a notch higher by focusing on the Open Source ecosystem in Asia, and more specifically, in India. |
```
Comment (nikhita) [inline README.md]: Please remove this conference since this does not have a CFP.

File: README.md [hunk 4]
```diff
@@ -10,14 +10,31 @@ To add a conference to the list, please [contribute](#contributing)!
...
+| [OSI](https://www.opensourceindia.in/) | 11-12 Oct | NIMHANS Convention Center, Bengaluru | Started as LinuxAsia in 2004, OSI has been at the helm of bringing together the Open Source industry and the community in the last 14 years. The 15th edition of OSI this year aims to take this event a notch higher by focusing on the Open Source ecosystem in Asia, and more specifically, in India. | 
 | [PyCon India](https://in.pycon.org/2018/) | 05-09 Oct | Hyderabad International Convention Centre (HICC), Hyderabad | The premier conference in India on using and developing the Python programming language. |
+| [IOT Summit](http://www.unicomlearning.com/IoT_Summit_Pune_2018/) | 28 Sep | Pune | The conference explores how the Internet of Things will re-shape our interactions with the real and virtual worlds in the coming years. Bringing together leading IoT companies, Start-Ups and the developer/ Professional community to exchange their ideas with the practical application and utilization of Internet of Things. |
```
Comment (nikhita) [inline README.md]: Please remove this conference since this does not have a CFP.

File: README.md [hunk 5]
```diff
@@ -10,14 +10,31 @@ To add a conference to the list, please [contribute](#contributing)!
...
 | [PyCon India](https://in.pycon.org/2018/) | 05-09 Oct | Hyderabad International Convention Centre (HICC), Hyderabad | The premier conference in India on using and developing the Python programming language. |
+| [IOT Summit](http://www.unicomlearning.com/IoT_Summit_Pune_2018/) | 28 Sep | Pune | The conference explores how the Internet of Things will re-shape our interactions with the real and virtual worlds in the coming years. Bringing together leading IoT companies, Start-Ups and the developer/ Professional community to exchange their ideas with the practical application and utilization of Internet of Things. | 
+| [DevOps Summit](http://www.unicomlearning.com/Devops_Summit_Pune_2018/) | 27 Sep | Pune | This conference brings together leading practitioner-organisations who have achieved Continuous Delivery and organisational transformation. |
```
Comment (nikhita) [inline README.md]: Please remove this conference since this does not have a CFP.

File: README.md [hunk 6]
```diff
@@ -10,14 +10,31 @@ To add a conference to the list, please [contribute](#contributing)!
...
+| [DevCon Summit](http://www.unicomlearning.com/DevCon_Bangalore_2018/) | 26-27 Sep | Le Meridien, Bengaluru | DevCon will be a practitioner-driven conference designed for technical team leads, architects, engineering directors, and project managers who influence innovation in their teams. Our conference and workshops invite you to get intro lessons (or advanced tips and tricks) on technologies like HTML 5, WebRTC, Dot Net ,Functional Programming , Go ,Scala ,Julia, Full-Stack Javascript Development, Mobile Web Design, Node.js, Data Science, and Distributed Computing to name a few. |
```
Comment (nikhita) [inline README.md]: Please remove this conference since this does not have a CFP.

File: README.md [hunk 7]
```diff
@@ -10,14 +10,31 @@ To add a conference to the list, please [contribute](#contributing)!
...
 | [JSFoo](https://jsfoo.in/2018/) | 13-14 Sep | Bengaluru | The eighth edition of India’s first JavaScript conference.|
+| [AI & ML](http://www.unicomlearning.com/Artificial_Intelligence_Machine_Learning_Bangalore_2018/) | 6 Sep | Hilton Bangalore EGL | Artificial Intelligence, Machine Learning and Sentiment Analysis are changing the way in which 'client focussed' services are offered. In particular Financial Organisations are creating and leveraging such innovation in the domain of wealth management.|
```
Comment (nikhita) [inline README.md]: Please remove this conference since this does not have a CFP.

File: README.md [hunk 8]
```diff
@@ -10,14 +10,31 @@ To add a conference to the list, please [contribute](#contributing)!
...
+| [Interspeech 2018](http://www.interspeech2018.org/index.html) | 2-6 Sep | Hyderabad International Convention Centre | Interspeech is the world’s largest and most comprehensive conference on the science and technology of spoken language processing |
```
Comment (nikhita) [inline README.md]: Please remove this conference since this does not have a CFP.

File: README.md [hunk 9]
```diff
@@ -10,14 +10,31 @@ To add a conference to the list, please [contribute](#contributing)!
...
+| [DevOps India Summit](https://www.xellentro.com/devops-india-summit-2018/) | 31 Aug - 1 Sep | Bengaluru Marriott Hotel Whitefield | DevOps India Summit 2018 is a conference for the Indian DevOps Community to hear for the first time the stalwarts of DevOps on what and how they have implemented DevOps in their complex organization and also know what the world is doing today in DevOps. |
```
Comment (nikhita) [inline README.md]: Please remove this conference since this does not have a CFP.

File: README.md [hunk 10]
```diff
@@ -10,14 +10,31 @@ To add a conference to the list, please [contribute](#contributing)!
...
 | [ODSC (One Data Science Conference) ](https://india.odsc.com/) | 30 Aug – 2 Sep | Hotel Lalit Ashok, Bengaluru  | Conference on AI and Data Science |
 | [ReactFoo-Delhi](https://reactfoo.in/2018-delhi/) | 18 Aug | TBA | React, alternatives to React, React Native and front-end engineering. |
 | [DevConf.in 2018](https://devconf.info/in) | 4-5 Aug | Christ University - Bengaluru | Annual Developers' Conference organised by Red Hat for local FOSS community |
 | [Deccan RubyConf 2018](https://www.deccanrubyconf.org/) | 4 Aug | Sheraton Grand Hotel, Pune | Pune extension of RubyConf India organized by same folks organizing RubyConf India |
+| [Blockchain Summit](http://www.unicomlearning.com/Blockchain_Summit_Mumbai-2018/) | 3 Aug | Mumbai | This conference brings together Block Chain technologists in the finance sector, regulators, industry commentators, computer science researchers and others involved in financial innovation; these experts explore some of the technology and the many applications of Block chain in Financial Services. |
```
Comment (nikhita) [inline README.md]: Please remove this conference since this does not have a CFP.

File: README.md [hunk 11]
```diff
@@ -10,14 +10,31 @@ To add a conference to the list, please [contribute](#contributing)!
...
 | [Container Conference](https://www.containerconf.in/) | 3-4 Aug | Radisson Blu (Park Plaza), ORR, Bangalore | Container Conference is a developer focused conference on Docker, Managed Kubernetes/Container Services, Service Mesh, Kubernetes |
+| [Data Visualisation Summit](http://www.unicomlearning.com/Data_Visualisation_Summit_Mumbai_2018/) | 2 Aug | The LaLiT, Mumbai | A one-day, single-track event focused exclusively contextualized to explore Visualisation tools and tips. Bringing together data enthusiasts who will help you rethink your data imagery to enable easier to interpret, management and action. |
```
Comment (nikhita) [inline README.md]: Please remove this conference since this does not have a CFP.

File: README.md [hunk 12]
```diff
@@ -10,14 +10,31 @@ To add a conference to the list, please [contribute](#contributing)!
...
+| [Call For Code Day](http://callforcodeindia.com/) | 20 Jul | BHAU Institute, Pune | Call for Code is more than a global challenge, it's an opportunity for developers to make a lasting change for social good. Unlike other hackathons and code challenges, IBM is providing developers with access to the best in IBM technology - Cloud, AI, IoT and more, to create applications that improve disaster preparedness, build resilient communities and safeguard the health and well-being of individuals and institutions. |
```
Comment (nikhita) [inline README.md]: Please remove all instances of this conference since all of them do not have a CFP.

**Comment by nikhita** [comment]
2018 is almost over, so I'll close this. Thanks for creating the PR, much appreciated!

**Pull request closed without merging**